app/src/store/actionCreators.test.js

```diff
@@ -0,0 +1,68 @@
+import { initialState } from './initialState';
+import { getMovies, deleteMovie, addMovie, updateMovie } from './actionCreators';
+import configureStore from 'redux-mock-store';

```

Comment:
u don't need store to test your actions.

u could mock `dispatch` and `getState`

Reply:
Really nice! Done